### ISSUE TITLE
Link gives invert error when using reactRouterLinkComponent

### ISSUE THREAD

**Issue by slawler-telus** [telus/tds-core#1303]
## Bug in Link when using reactRouterLinkComponent

if `reactRouterLinkComponent` is set, then the following error occurs:
```
Warning: Received `false` for a non-boolean attribute `invert`.

If you want to write it to the DOM, pass a string instead: invert="false" or invert={value.toString()}.

If you used to conditionally omit it with invert={condition && value}, pass invert={condition ? value : undefined} instead.
```

## Reproduction Steps
To produce the error:
```
import { Link as ReactRouterLink } from 'react-router-dom'
import Link from '@tds/core-link'

      const isInverted = variant === 'inverted' ? 'true' : undefined
      const useRouterComponent = relative ? ReactRouterLink : undefined
      return (
        <Link
          invert={isInverted}
          href={!relative ? linkUrl : null}
          to={relative ? linkUrl : null}
          reactRouterLinkComponent={useRouterComponent}
        >
          Link Label
        </Link>
      )
```

Same code with no error:
```
// import { Link as ReactRouterLink } from 'react-router-dom'
import Link from '@tds/core-link'

      const isInverted = variant === 'inverted' ? 'true' : undefined
      // const useRouterComponent = relative ? ReactRouterLink : undefined
      return (
        <Link
          invert={isInverted}
          href={!relative ? linkUrl : null}
          to={relative ? linkUrl : null}
        >
          Link Label
        </Link>
      )
```

## Workaround details

Don't use `useRouterComponent`

## Meta

- TDS component version: Tried with both `2.0.10` and `2.1.2`
- Willing to develop solution: Yes/No
- High impact: No, just console errors. It still renders file

## Screenshots
![image](https://user-images.githubusercontent.com/42187593/67722666-ff96e300-f996-11e9-8e03-8a085adad351.png)

Code where it happens: https://github.com/telus/my-telus-rewards/blob/staging/app/src/ui/components/MultiLink/index.jsx#L102

**Comment by marcod1419** [comment]
(For whoever picks this up) This is most likely an issue with `Link` incorrectly passing the `invert` prop. Perhaps start by looking at how props change via react dev tools when using a router component.

**Comment by theetrain** [comment]
This has been resolved in [`@tds/core-link@2.1.3`](https://github.com/telus/tds-core/releases/tag/%40tds%2Fcore-link%402.1.3)
